cluster/plan.go

```diff
@@ -505,7 +505,7 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 	var Binds []string
 
 	if c.IsCRIDockerdEnabled() {
-		CommandArgs["container-runtime"] = "remote"
+		//CommandArgs["container-runtime"] = "remote" https://v1-26.docs.kubernetes.io/docs/reference/command-line-tools-reference/kubelet/
```

Comment:
Does `container-runtime` default to `remote` for <v1.27 versions? If not, we should add a separate check for >=v1.27. Need to make sure we don't break older versions by removing this field.

Reply:
No I don't think it defaults to remote for <v1.27 versions. Will add a separate check here.